channeldb/channel.go

```diff
@@ -657,6 +657,15 @@ type OpenChannel struct {
 	// received within this channel.
 	TotalMSatReceived lnwire.MilliSatoshi
 
+	// InitialLocalBalance is the balance we have during the channel
+	// opening. When we are not the initiator, this value represents the
+	// push amount.
+	InitialLocalBalance lnwire.MilliSatoshi
```

Comment:
What's the purpose of this given that it can be derived from the opening capacity and push amount existence?

Reply:
I don't think we store the push amount in db?

Reply:
Ah yeah you're right, we just map this to what the local balances should be when processing the reservations. 

However I think it's obtainable from the SCB itself? In either case, not qualms to making this small piece of data easily accessible via persistence.